server/testutils/factories/cas1SpaceBookingSummary.ts

```diff
@@ -50,6 +50,10 @@ class Cas1SpaceBookingSummaryFactory extends Factory<Cas1SpaceBookingSummary> {
       isNonArrival: true,
     })
   }
+
+  cancelled() {
+    return this.upcoming().params({})
+  }
```

Comment:
There is no way to indicate cancelled placement in the `Cas1PlacementSummary` now, so this is a bit weird.